docs/apache-airflow/concepts/taskflow.rst

```diff
@@ -182,6 +182,15 @@ for deserialization ensure that ``deserialize(data: dict, version: int)`` is spe
 
   Note: Typing of ``__version__`` is required and needs to be ``ClassVar[int]``
 
+
+Sensors and the TaskFlow API
+--------------------------------------
+
+.. versionadded:: 2.5.0
+
+For an example of writing a Sensor using the TaskFlow API, see
+:ref:`Using the TaskFlow API with Sensor operators <taskflow/task_sensor_example>`.
```

Comment:
I feel like we should have some actual content here instead of just linking over?

Reply:
What in particular would fit in well here? I'd rather not take the same content from over there since it seems to make sense there.